shared/chat/conversation/messages/text/index.js

```diff
@@ -9,15 +9,24 @@ export type Props = {
   mentionsAt: Types.MentionsAt,
   mentionsChannel: Types.MentionsChannel,
   mentionsChannelName: Types.MentionsChannelName,
+  message: Types.MessageText,
   text: string,
   type: 'error' | 'pending' | 'sent',
 }
 
-const MessageText = ({text, type, isEditing, mentionsAt, mentionsChannel, mentionsChannelName}: Props) => {
+const MessageText = ({
+  text,
+  type,
+  isEditing,
+  mentionsAt,
+  mentionsChannel,
+  mentionsChannelName,
+  message,
+}: Props) => {
   const markdown = (
     <Kb.Markdown
       style={getStyle(type, isEditing)}
-      meta={{mentionsAt, mentionsChannel, mentionsChannelName}}
+      meta={{mentionsAt, mentionsChannel, mentionsChannelName, message}}
```

Comment:
Why pass the entire message down   We extract at already

Reply:
Why not? The entire message is a prop to `SendPaymentPopup`.

Reply:
That thing should be connected and just get the id it needs

Reply:
When we discussed on Friday we weee talking about the payment id and that's it

Reply:
`SendPaymentPopup` is something that already exists, I didn't want to change it too much, since it already seems to work fine. It is connected, but needs the message to render itself when it is being used on a normal Stellar send message. It is also convenient for getting things out, like the author and sending device, which is what this component uses it for.

Reply:
If that's the case we should do what we're doing with mentions. Now we're sending the whole message down and yet peeling off several parts

Reply:
Why? It seems like the right move is to just send the whole thing down and mentions pull off what it needs. Otherwise now I have some dependency in the text component on what some downstream things the markdown parser might generate need. Seems better to send the whole message down and let those components pick out what they want. Is there some performance thing I am missing here?

Reply:
Did you change should to shouldnt? If so, I didn't want to refactor mentions as part of this, I can pick that up if we change how that works in general (which is to use the method I am using for these Stellar messages).

Reply:
Maybe I'm misunderstanding. We're sending a message and mentions. The mentions are in the message also. They come from the message

Reply:
I did as that was a typo

Reply:
Yeah see above, I didn't want a whole bunch of changes to mentions in this PR, seems appropriate to fix that in some other change.